core-api-library/library/src/main/java/net/gini/android/core/api/requests/SafeApiRequest.kt

```diff
@@ -0,0 +1,39 @@
+package net.gini.android.core.api.requests
+
+import org.json.JSONException
+import org.json.JSONObject
+import retrofit2.Response
+import java.util.concurrent.CancellationException
+
+private const val ERROR_MESSAGE_KEY = "message"
+private const val ERROR_CODE_KEY = "code"
+
+object SafeApiRequest {
+    @Throws(ApiException::class)
```

Comment:
Please also add the `CancellationException` to make it obvious that it can be thrown as well.